SampleApp/TangramVC.swift

```diff
@@ -8,16 +8,30 @@
 
 import UIKit
 import TangramMap
-class TangramVC:  MapViewController{
+class TangramVC:  MapViewController, MapMarkerSelectDelegate {
```

Comment:
Agreed. I'll open an issue as it's been mentioned before by some others on our team. This is just our sample app so I've been de-prioritizing refactoring it over other tasks :)